packages/build/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+var log = require( 'fancy-log' );
+var gulp = require( 'gulp' );
+var gulpif = require( 'gulp-if' );
+var minify = require( 'gulp-minify' );
+var PluginError = require( 'plugin-error' );
+var prepend = require( 'gulp-append-prepend' );
+var rename = require( 'gulp-rename' );
+var saveLicense = require( 'uglify-save-license' );
+var sourcemaps = require( 'gulp-sourcemaps' );
+var through = require( 'through2' );
+// import webpack from 'webpack';
+
+// function getWebpackConfig() {
+// 	return require( './../webpack.config.js' );
+// }
+
+// export const watch = function() {
+// 	const config = getWebpackConfig();
+
+// 	return webpack( config ).watch(
+// 		100,
+// 		onBuild.bind( this, function( error ) {
+// 			if ( error ) {
+// 				log( error );
+// 				return;
+// 			}
+// 		} )
+// 	);
+// };
+
+// gulp.task( 'react:modules', function( done ) {
+// 	const config = getWebpackConfig();
+// 	return webpack( config ).run( onBuild.bind( this, done ) );
+// } );
+
+module.exports = {
+	build_module_assets: function( done, err, stats ) {
```

Comment:
👋  I know this is a work in progress, but @Automattic/luna has spent a significant amount of time working on builds and we’d love for that experience to inform the road ahead.

We're just a ping away if you’d like to put our heads together 🙂